### DISCUSSION TITLE
What does the first u in uutils means?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [uutils/coreutils#5180] [Q&A]
Just curious.

**Comment by tertsdiepraam** [comment]
The name was decided on long before I even heard of the project, but I always understood it as "universal" (as in cross-platform). But it's not really an abbreviation, I suppose, because we never write "universal utils". So, I think you could apply any meaning you want. 😄 Maybe "you" or "updated" or "ultra" or "ubiquitous" or "united" or "unified" etc.

Or "ugly" if you strongly prefer the BSD style of utils over the GNU style 😉

The reason that I believe it's "universal" is because this was the original description in the README:
> uutils is an attempt at writing universal (as in cross-platform) CLI utils in [Rust](http://rust-lang.org). This repo is to aggregate the GNU coreutils rewrites.
>
>Why?
> ----
>
> Many GNU, linux and other utils are pretty awesome, and obviously
> [some](http://gnuwin32.sourceforge.net) [effort](http://unxutils.sourceforge.net)
> has been spent in the past to port them to windows. However those projects
> are either old, abandonned, hosted on CVS, written in platform-specific C, etc.
>
> Rust provides a good platform-agnostic way of writing systems utils that are easy
> to compile anywhere, and this is as good a way as any to try and learn it.

Which was added in the second commit of the project all the way back in 2013.

**Reply by YousufSSyed** [reply]
Good answer, I'll wait for more.
